dvc/repo/metrics/show.py

```diff
@@ -188,9 +188,14 @@ def _collect_metrics(repo, path, recursive, typ, xpath, branch):
         if not o.metric:
             continue
 
-        # NOTE this case assumes that metric has been modified with
-        # `dvc metric modify -t {} -x {}`, so stage file should contain
-        # info what type metric is and what is its xpath
+        # NOTE this case assumes:
```

Comment:
Single-entry list is not needed :) 
```suggestion
        # NOTE this case assumes that typ has not been provided in CLI call and one of the following cases:
```